grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -533,9 +533,14 @@ class UrlMappings {
             action = [POST: "validateInvoiceItem"]
         }
 
+        "/api/prepaymentInvoices/$id/invoiceItems" {
```

Comment:
I'm not generally a fan of the camelcase URIs. I usually try to stick to a feature/sub-feature/entity-based system with something like: `/api/invoices/prepayment/$id/items` or `/api/prepayment-invoices/$id/invoice-items`. Up to you though

Reply:
We don't have a written convention for these yet (camel vs kebab case), hence I am following for now what we have in other cases (ex. `stockMovements`, `partialReceiving`, etc)